.github/scripts/github_metrics.py

```diff
@@ -0,0 +1,91 @@
+"""
+Submit metrics returns "Payload accepted" response
+"""
+
+from datetime import datetime
+from datetime import timedelta
+from datadog_api_client.v1 import ApiClient, Configuration
+from datadog_api_client.v1.api.metrics_api import MetricsApi
+from datadog_api_client.v1.model.metrics_payload import MetricsPayload
+from datadog_api_client.v1.model.point import Point
+from datadog_api_client.v1.model.series import Series
+import requests
+
+
+github_url="https://api.github.com/repos/twilio/twilio-cli/releases"
+response = requests.get(github_url)
+output=response.json()
+rpm_count = deb_count = exe_count = pkg_count = 0
+
+for release in output:
+    for platform_exe in release['assets']:
+        if(platform_exe['name'][-3:]=="rpm"):
+            rpm_count+=platform_exe['download_count']
+        elif(platform_exe['name'][-3:]=="deb"):
+            deb_count+=platform_exe['download_count']
+        elif(platform_exe['name'][-3:]=="exe"):
+            exe_count+=platform_exe['download_count']
+        elif(platform_exe['name'][-3:]=="pkg"):
+            pkg_count+=platform_exe['download_count']
+
+download_count_rmp_float=float(rpm_count)
+download_count_deb_float=float(deb_count)
+download_count_exe_float=float(exe_count)
+download_count_pkg_float=float(pkg_count)
+
+
+body_rpm = MetricsPayload(
+    series=[
+        Series(
+            metric="github.rpm.downloads_count",
+            type="count",
+            points=[Point([datetime.now().timestamp(), download_count_rmp_float])],
+            tags=["test:github"],
+        )
+    ]
+)
+
+body_deb = MetricsPayload(
+    series=[
+        Series(
+            metric="github.deb.downloads_count",
+            type="count",
+            points=[Point([datetime.now().timestamp(), download_count_deb_float])],
+            tags=["test:github"],
+        )
+    ]
+)
+
+body_exe = MetricsPayload(
+    series=[
+        Series(
+            metric="github.exe.downloads_count",
+            type="count",
+            points=[Point([datetime.now().timestamp(), download_count_exe_float])],
+            tags=["test:github"],
+        )
+    ]
+)
+
+body_pkg = MetricsPayload(
+    series=[
+        Series(
+            metric="github.pkg.downloads_count",
+            type="count",
+            points=[Point([datetime.now().timestamp(), download_count_pkg_float])],
+            tags=["test:github"],
+        )
+    ]
+)
+
+configuration = Configuration()
+with ApiClient(configuration) as api_client:
+    api_instance = MetricsApi(api_client)
+    response1 = api_instance.submit_metrics(body=body_rpm)
+    print("Github rpm metrics submission status: ",response1)
```

Comment:
Is there a log generated with these statements or is there a sample response that we can see?

Reply:
These logs just make it easier for us to debug and find out which step failed. When successful, the response is just a 200 status. These logs will just help us save some time

Reply:
Any sample logs available?

Reply:
https://github.com/shamantraghav/twilio-cli/actions/runs/1704804569

Reply:
Okay, same as the one mentioned earlier.👍